### PR TITLE
Adding Uruguay's digital political party

### PR DESCRIPTION
**Pull request by leog**
Partido Digital is the first Uruguayan Open Source digital-based political party where transparency and participation are crucial.

### CODE REVIEW

**Comment by nschonni** [comment]
Think this would probably be more civic tech since it appears to be a political rather than a government (public service) organization.

**Comment by leog** [comment]
Wow, missed that option. Going for the Civic Hackers then, good catch @nschonni 👍

**Comment by leog** [comment]
There you go @nschonni!

File: _data/civic_hackers.yml [hunk 1]
```diff
@@ -120,6 +120,7 @@ Civic Hackers:
   - unitedstates
   - votinginfoproject
   - yourmapper
+  - PartidoDigital
```
Comment (nschonni) [inline _data/civic_hackers.yml]: Nit: These files are alphabetical, so it's better to add it above

**Review by nschonni — CHANGES_REQUESTED** [review]
LGTM, just take a look at the ordering in the file

**Comment by leog** [comment]
Done @nschonni 🙂

**Comment by leog** [comment]
Squished commits to have a clear history @nschonni.

**Comment by nschonni** [comment]
@jbjonesjr LGTM

**Comment by jbjonesjr** [comment]
:ship: Thanks @leog. Very cool by the way. :heart: that transparency

**Pull request merged**